### ISSUE TITLE
Historic Mountain View boundaries

### ISSUE THREAD

**Issue by 1ec5** [codeforsanjose/OSM-SouthBay#33]
The City of Mountain View maintains a GIS layer of [annexation areas](https://data-mountainview.opendata.arcgis.com/datasets/MountainView::annexations/). Each area comes with two dates: the date on which the annexation ordinance was filed by the city, and the date on which it was enrolled into the county’s resolution book. We can create boundary relations based on these areas, similar to San José in #28, and tag them with one of the dates. Mountain View doesn’t border San José, so boundary conflation won’t be necessary just yet.

**Comment by 1ec5** [comment]
@jeffreyameyer imported this dataset last August. However, the tags on some of the features need to be redone. For example, [way 199,589,446](https://www.openhistoricalmap.org/way/199589446) has lots of junk tags autogenerated from ArcGIS. All the ways need to be converted into boundary relations with `inner` and `outer` rings and added to a chronology relation.

**Comment by jeffreyameyer** [comment]
Yes - some of those tags are junk (not sure why I left the `ShapeST*` - I can clean those up. Some of the other metadata may be useful. I'll do a quick cleanup.

**Comment by 1ec5** [comment]
> All the ways need to be converted into boundary relations with `inner` and `outer` rings

I documented an almost trivial process for this conversion as part of the [time zone import](https://wiki.openstreetmap.org/wiki/OpenHistoricalMap/Projects/United_States/Time_zone_boundaries#Bulk_import). We need to eliminate [these ways](https://overpass-turbo.openhistoricalmap.org/?Q=%2F*%0AThis%20has%20been%20generated%20by%20the%20overpass-turbo%20wizard.%0AThe%20original%20search%20was%3A%0A%E2%80%9Cwikidata%3DQ486860%20and%20type%3Away%20global%E2%80%9D%0A*%2F%0A%5Bout%3Ajson%5D%5Btimeout%3A25%5D%3B%0A%2F%2F%20gather%20results%0Away%5B%22wikidata%22%3D%22Q486860%22%5D%3B%0A%2F%2F%20print%20results%0Aout%20geom%3B&C=37.412126%3B-122.081256%3B13) in order to conflate the Mountain View boundaries with those of neighboring Palo Alto.
